### PR TITLE
Closes #448 R-B:Not allowing text behind the security icon be selectable

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes a changelog entry or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

Reference Browser issue https://github.com/mozilla-mobile/reference-browser/issues/448

### CODE REVIEW

**Comment by Amejia481** [comment]
After doing many tests, I realized that AppCompatImageView allows text behind it to be selectable, when it doesn't have click listener set. I believe in the future we are going to expose the click event for consumers know, when the security icon was clicked, but right now, I'm not sure if this the right moment, for this reason I prevent the issue for keep happening.

File: components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt [hunk 1]
```diff
@@ -148,6 +153,9 @@ internal class DisplayToolbar(
             field = value
         }
 
+    // Optional callback to be call when the icon.
+    internal var onIconClicked: (() -> Unit)? = null
+
```
Comment (pocmo) [inline components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt]: This one seems unneeded?

Comment (Amejia481) [reply components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt]: Sorry a leftover from some previous experiments :P

Comment (pocmo) [reply components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt]: Why are you not onboarding?? :P

**Comment by Amejia481** [comment]
PR updated :)

**Pull request merged**